pkg/cmd/dev/lint.go

```diff
@@ -44,6 +44,10 @@ func (d *dev) lint(cmd *cobra.Command, commandLine []string) error {
 	timeout := mustGetFlagDuration(cmd, timeoutFlag)
 	short := mustGetFlagBool(cmd, shortFlag)
 
+	if len(pkgs) > 0 && pkgs[0] == "short" {
```

Comment:
Nit: Maybe we should trigger an error if *any* of the `pkgs` are `"short"`?

Reply:
Yep. It makes sense to extend it to cases like `./dev lint //pkg/foo/bar short`.